### PR TITLE
Remove '.only' from 'analyzer' test

### PR DESCRIPTION
**Pull request by vilchik-elena**
Also fixing one test (one more token to highlight is `// nosonar` comment)

### CODE REVIEW

File: eslint-bridge/tests/analyzer.test.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ describe('#analyzeJavaScript', () => {
     expect(result.highlights).toHaveLength(11);
   });
 
-  it.only('should analyze both main and test files', () => {
```
Comment (yassin-kammoun-sonarsource) [inline eslint-bridge/tests/analyzer.test.ts:97]: I am wondering whether we should have a rule reporting on `only`-like unit tests. It's likely a mistake. WDYT ?

**Review by yassin-kammoun-sonarsource — APPROVED** [review]
LGTM. We should probably merge #2858 first.

File: eslint-bridge/tests/analyzer.test.ts (hunk 1 above)
Comment (vilchik-elena) [reply eslint-bridge/tests/analyzer.test.ts:97]: we definitely should :) Is there such thing in mocha btw?

Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/tests/analyzer.test.ts:97]: Yes, there is: see https://mochajs.org/#exclusive-tests

Comment (saberduck) [reply eslint-bridge/tests/analyzer.test.ts:97]: I think this is great rule idea

Comment (vilchik-elena) [reply eslint-bridge/tests/analyzer.test.ts:97]: https://github.com/SonarSource/SonarJS/issues/2861

**Pull request merged**